doc/source/GettingStarted/TutorialAssemblingTheGafferBot/index.md

```diff
@@ -176,62 +176,62 @@ When a scene node computes a scene, scene data passes through it like so:
 
 ### Plugs ###
 
-The inputs and outputs of a node are called **plugs**, and are represented in the _Graph Editor_ as colored circles around the node's edges.
+The inputs and outputs of a node are called **plugs**, and are represented in the Graph Editor as colored circles around the node's edges.
 
 <!-- TODO: add close-up of node plugs ![](images/nodePlug.png "A node's plug") -->
 
-For your two nodes to occupy the same scene (and later render together), you will need to combine them into a single scene. You can connect both of their output plugs to a Group node, and you can also rearrange the nodes to better visually represent the data flow in the graph.
+For your two nodes to occupy the same scene (and later render together), you will need to combine them into a single scene. You can connect both of their Out plugs to a Group node, and you can also rearrange the nodes to better visually represent the data flow in the graph.
```

Comment:
`Out`  plugs aren't capitalised in the UI:

![image](https://user-images.githubusercontent.com/896779/72746377-e07cab80-3ba9-11ea-9b64-879a61ae2b8b.png)